\hypertarget{classpip_1_1__internal_1_1exceptions_1_1DistributionNotFound}{}\doxysection{pip.\+\_\+internal.\+exceptions.\+Distribution\+Not\+Found Class Reference}
\label{classpip_1_1__internal_1_1exceptions_1_1DistributionNotFound}\index{pip.\_internal.exceptions.DistributionNotFound@{pip.\_internal.exceptions.DistributionNotFound}}


Inheritance diagram for pip.\+\_\+internal.\+exceptions.\+Distribution\+Not\+Found\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=217pt]{classpip_1_1__internal_1_1exceptions_1_1DistributionNotFound__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for pip.\+\_\+internal.\+exceptions.\+Distribution\+Not\+Found\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=217pt]{classpip_1_1__internal_1_1exceptions_1_1DistributionNotFound__coll__graph}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when a distribution cannot be found to satisfy a requirement\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
inventaria\+\_\+env/lib/python3.\+10/site-\/packages/pip/\+\_\+internal/exceptions.\+py\end{DoxyCompactItemize}
